content/en/docs/tasks/administer-cluster/subpath-env.md

```diff
@@ -0,0 +1,54 @@
+---
+title: Using subPath with expanded environment variables 
+content_type: task
+---
```

Comment:
Is this task page related to the updates to the Volumes page?

Reply:
I think it's related to the concept of "refactoring" the volumes page. In response to feedback, I broke those out into separate pages.